src/scoreboard/controllers/initscoreboardcontroller.js

```diff
@@ -0,0 +1,167 @@
+import Controller from './controller';
+
+export default class InitGameController extends Controller
+{
+    /**
+     * Displays the initial state
+     *
+     * @override
+     */
+    view()
+    {
+        [
+            //
```

Comment:
This doesn't appear to do much

Reply:
Once you remove this part I'll merge it in, as it seems more than fine for this iteration of the game.